src/main/java/org/prebid/server/proto/openrtb/ext/request/rubicon/VideoType.java

```diff
@@ -0,0 +1,15 @@
+package org.prebid.server.proto.openrtb.ext.request.rubicon;
+
+public enum VideoType {
```

Comment:
No need for separate `enum` since we have only one value so far.
Please, use plaing `rewarded` string where needed.